src/kixi/file_size.clj

```diff
@@ -0,0 +1,96 @@
+(ns kixi.file-size
+  (:require [amazonica.aws.s3 :as s3]
+            [kixi.maws :refer [witan-prod-creds]]
+            [clojure.java.io :as io]
+            [clojure.string :as string]
+            [cheshire.core :as json])
+  (:import [java.io ByteArrayInputStream File InputStream]))
+
+(def dynamo-backup-bucket "prod.witan-dynamo-backup")
+
+(def dynamo-backup-prefix "kixi.datastore/2017-10-18-04-26-10/prod-kixi.datastore-metadatastore-")
+
+(def backup-local-dir "metadatastore-backup")
+
+(def credentials (assoc (witan-prod-creds) :client-config {:max-connections 50
+                                                           :connection-timeout 5000
+                                                           :socket-timeout 5000}))
+
+(defn download-backup
+  [s3-object-summary]
+  (let [^File local-file (io/file backup-local-dir (last (string/split (:key s3-object-summary) #"/")))]
+    (when-not (.exists local-file)
+      (do (.createNewFile local-file)
+          (let [s3-object (s3/get-object credentials
+                                         :bucket-name dynamo-backup-bucket
+                                         :key (:key s3-object-summary))]
+            (with-open [^InputStream in (:input-stream s3-object)]
+              (io/copy in local-file)))))
+    local-file))
+
+(defn decode-file
+  [file-name]
+  (with-open [in (io/input-stream file-name)]
+    (json/parse-stream (io/reader in) true)))
+
+(defn parse-int
+  [item]
+  (if-let [s ((comp :N :kixi.datastore.metadatastore_size-bytes) item)]
+    (Integer/parseInt s)
+    (do
+      (prn item)
```

Comment:
Assume this a debug `prn`

Reply:
This is an error report. When the thing runs it should be silent, any output is bad.